env/templates/700-hook-ing.yaml

```diff
@@ -1,3 +1,4 @@
+{{- if .Values.lighthouse.enabled }}
```

Comment:
This should probably be a .Values.prow.enabled check as hook is a Prow component rather than a lighthouse component

Reply:
Fixed